packages/types/src/core/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./db"
```

Comment:
There is already the top level `sdk` package inside types. Functionally that is the `core` module, for everything that isn't an api request or a document